libvmi/os/windows/windows.h

```diff
@@ -23,6 +23,10 @@
 
 #include "private.h"
 
+#define LOWEST_KERNEL_ADDRESS_64BIT 0xffff800000000000
+#define LOWEST_KERNEL_ADDRESS_32BIT 0x80000000
```

Comment:
Right. I made this correction in my win10 pr already btw. I'll have to check if there are any other situations where the kernel va is used but I think it's really only used during a sanity check (in which case its a bit of an overkill to have a separate config entry).